packages/neos-ui/src/Sagas/Browser/index.js

```diff
@@ -0,0 +1,15 @@
+import {takeEvery} from 'redux-saga';
+
+import {actionTypes} from '@neos-project/neos-ui-redux-store';
+
+function * watchContentURIChange() {
+    yield * takeEvery(actionTypes.UI.ContentCanvas.SET_CONTEXT_PATH, function * reflectChangeInAddressBar(action) {
+        const {contextPath} = action.payload;
+
+        yield history.replaceState({}, '', `?node=${contextPath}`);
```

Comment:
We might want to use the [`call`](https://redux-saga.js.org/docs/api/#callfn-args) API from redux saga here to enhance the testability, wdyt @dimaip?

Reply:
Agree, if somebody was into writing tests for this he could change it. But since this saga is just 2 lines of code with one execution path I had no intentions to tests it, rather fix an obvious bug with the least amount of changes :)

Reply:
Btw, I get super excited when I see comments from you in this repo, good to see you back ;)

Reply:
Ah, and I didn't write that code, I just reverted the commit I deleted before with one small change.